src/client/components/table-of-contents/table-of-contents.vue

```diff
@@ -14,5 +15,78 @@
 <script>
 export default {
   props: ['items'],
+  data () {
+    return {
+      currentSection: this.items[0].slug
```

Comment:
We typically use `activeItem: this.items[0]`. I think the mental modal of having an active item and comparing other items with it is easier.